documentation/en/default-lotus-miner-config.toml

```diff
@@ -666,9 +666,9 @@
   # env var: LOTUS_SEALING_TERMINATEBATCHWAIT
   #TerminateBatchWait = "5m0s"
 
-  # UseSyntheticPoRep will reduce the amout of data held on disk in the WaitSeed phase to 32GiB
-  # at the cost of having to precompute the synthetic challenges.
-  # TODO: put estimated costs here
+  # UseSyntheticPoRep, when set to true, will reduce the amount of cache data held on disk after the completion of PreCommit 2 to 11GiB.
+  # This is achieved by precomputing "synthetic" challenges from CommR.
```

Comment:
It’s not computed from commr, it’s reading a lot of challenges from the whole cache, and later we’re picking a few from that set in C1

Reply:
Yeah, I can probably update the wording to "Based on the CommR, the Storage Providers generates a list of "Synthetic Challenges" as written in the [FIP itself](https://fips.filecoin.io/FIPS/fip-0059.html) says that:

Also happy to discard that sentence, as I´m unsure what value that sentence actually brings to someone reading the config.